docs/tutorials/05_benchmarking.ipynb

```diff
@@ -20,33 +20,47 @@
    "cell_type": "code",
    "execution_count": 1,
    "metadata": {},
-   "outputs": [],
+   "outputs": [
+    {
+     "ename": "ModuleNotFoundError",
+     "evalue": "No module named 'benchmark_database'",
+     "output_type": "error",
+     "traceback": [
+      "\u001b[0;31m---------------------------------------------------------------------------\u001b[0m",
+      "\u001b[0;31mModuleNotFoundError\u001b[0m                       Traceback (most recent call last)",
+      "\u001b[0;32m<ipython-input-1-ceac042ac17e>\u001b[0m in \u001b[0;36m<module>\u001b[0;34m\u001b[0m\n\u001b[1;32m      5\u001b[0m \u001b[0;32mfrom\u001b[0m \u001b[0mIPython\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mdisplay\u001b[0m \u001b[0;32mimport\u001b[0m \u001b[0mVideo\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m      6\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m----> 7\u001b[0;31m \u001b[0;32mfrom\u001b[0m \u001b[0mbenchmark_database\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mload\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mbenchmark_database\u001b[0m \u001b[0;32mimport\u001b[0m \u001b[0mBenchmarkDatabase\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m      8\u001b[0m \u001b[0;32mfrom\u001b[0m \u001b[0mbenchmark_database\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mserialization\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mdatabase_serializer\u001b[0m \u001b[0;32mimport\u001b[0m \u001b[0mDatabaseSerializer\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m      9\u001b[0m \u001b[0;32mfrom\u001b[0m \u001b[0mbark\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mbenchmark\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mbenchmark_runner\u001b[0m \u001b[0;32mimport\u001b[0m \u001b[0mBenchmarkRunner\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mBenchmarkConfig\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mBenchmarkResult\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
+      "\u001b[0;31mModuleNotFoundError\u001b[0m: No module named 'benchmark_database'"
+     ]
+    }
+   ],
    "source": [
     "import os\n",
     "import matplotlib.pyplot as plt\n",
     "from IPython.display import Video\n",
     "\n",
-    "from benchmark_database.load.benchmark_database import BenchmarkDatabase\n",
-    "from benchmark_database.serialization.database_serializer import DatabaseSerializer\n",
-    "from bark.benchmark.benchmark_runner import BenchmarkRunner, BenchmarkConfig, BenchmarkResult\n",
-    "from bark.benchmark.benchmark_analyzer import BenchmarkAnalyzer\n",
+    "from load.benchmark_database import BenchmarkDatabase\n",
+    "from serialization.database_serializer import DatabaseSerializer\n",
```

Comment:
Something wrent wrong when merging this. This notebook still uses the old bark structure. Please ensure that all other notebooks still work-